pkg/cmd/apps.go

```diff
@@ -27,7 +27,7 @@ func MakeApps() *cobra.Command {
 		SilenceUsage: true,
 	}
 
-	install.PersistentFlags().String("kubeconfig", "kubeconfig", "Local path for your kubeconfig file")
```

Comment:
Why did you remove the default value?

Reply:
The default value was populated into `command.Flags().GetString("kubeconfig")` and not a valid value since it is not a path

Reply:
I did not set the default value to `~/.kube/config` because of possible cross platform issues and kubectl respecting the ENV variable KUBECONFIG by default so there should be no need to set it via k3sup as `--kubeconfig`

Reply:
👍